recognition/advanced_tree_search.py

```diff
@@ -176,6 +176,8 @@ def __init__(
         use_gpu=False,
         rtf=10,
         mem=4,
+        cpu=1,
```

Comment:
independent of keeping the old default.. shouldn't it be at least 2 cpus used (at least when using TF models) because RASR will anyway use more than one cpu?

Reply:
I agree with @michelwi

Reply:
Most of the legacy RASR setups can actually just make use of a single CPU. Only with the correct Tensorflow settings and MKL support it can actually use more. I will update the cpu count once I add those correct flags to the job, but I would not do it before.

Reply:
I remember that the AdvancedTreeSearchJobs regularly ignored the `MKL_NUM_THREADS` and `OMP_NUM_THREADS` directives and still used 1.5 cores worth of computation causing overload on the nodes. maybe @christophmluscher remembers and can comment.

Reply:
I think what you mean is that Tensorflow spawned two threads independend of those settings. And you are right about this, tensorflow can (and will) do that in some scenarios (Tensorflow and MKL threads are something different). But afaik this can not "escape" the CPU assignment limitation.

Reply:
I disagree. TF can escape the SGE limit. We see this regularly for AdvancedTreeSearchJobs with Neural AM aka TF. This is not much but we get high load warnings if this is not set correctly.

Reply:
A SGE Job with `cpu=1` uses then for example `cpu=1.1`

Reply:
Wait, I remember I wanted to change this at some point but Eugen stopped this...

Reply:
Also @christophmluscher the jobs from your setup were queued with cpu=1 as well... so why is this wrong now?

Reply:
because for a GMM-HMM decoding the default is `1` and this is fine. only for decoding with neural AM this is a problem. So only with RASR+TF this occurs. so when we see this issue occur we ask the users to increase manually. instead of just increasing for everybody.

I would leave at `1`

Reply:
ok, If you think so, then I will concur